### PR TITLE
Creat pr-assignment-7

### PR DESCRIPTION
**Pull request by rukshonarad**

### CODE REVIEW

**Review by hulkarsharif — APPROVED** [review]

File: assignment/assignment-7/assignment-7.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            <label for="monthInput">Preferred Meeting Month</label>
+            <input type="month" id="monthInput" name="PreeferredMonth" />
+
+            <hr />
+
+            <h2 style="text-align: center">Search</h2>
+            <label for="serchInput"
+                >Search for articles <a href="https://google.com"></a
+            ></label>
+            <input type="search" id="serchInput" name="searchArticles" />
```
Comment (hulkarsharif) [inline assignment/assignment-7/assignment-7.html:167]: typo mistake, pls correct

Comment (rukshonarad) [reply assignment/assignment-7/assignment-7.html:167]: Good catch. Will fix

File: assignment/assignment-7/assignment-7.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                type="tel"
+                placeholder="Phone number"
+                id="phoneInput"
+                name="phoneNumer"
+                pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"
+                autofocus
+            />
+            <br />
+
+            <label for="dobInput">Date of birthd</label>
```
Comment (hulkarsharif) [inline assignment/assignment-7/assignment-7.html:48]: correct the text birth

**Review by hulkarsharif — COMMENTED** [review]
just typo mistakes

**Pull request closed without merging**